tests/tape/tapes/test_cv_param_shift.py

```diff
@@ -814,7 +815,7 @@ def test_squeezed_mean_photon_variance(self, tol):
         assert np.allclose(grad, expected, atol=tol, rtol=0)
 
     def test_displaced_thermal_mean_photon_variance(self, tol):
-        """Test gradient of the photon variance of a displaced thermal state"""
+        """Test gradient of the photon qml.variance of a displaced thermal state"""

```

Comment:
```suggestion
        """Test gradient of the photon variance of a displaced thermal state"""
```
Actually this could probably be fixed much more easily by replacing `qml.variance` everywhere with `variance` :thinking: